examples/README.txt

```diff
@@ -5,5 +5,10 @@ General examples
 
 Introductory examples.
 
+These examples use data from standard machine learning libraries to show 
```

Comment:
I think we want to say something like: 

    These examples use data from the UCI machine learning data set repository to show...